www/include/configuration/configObject/host/listHost.php

```diff
@@ -94,9 +94,9 @@
 } else {
     //restoring saved values
     $search = $centreon->historySearch[$url]['searchH'] ?? null;
-    $poller = (int)$centreon->historySearch[$url]["poller"] ?? 0;
-    $hostgroup = (int)$centreon->historySearch[$url]["hostgroup"] ?? 0;
-    $template = (int)$centreon->historySearch[$url]["template"] ?? 0;
+    $poller = (int)($centreon->historySearch[$url]["poller"] ?? 0);
```

Comment:
as "0" is an integer, is it usefull to add the parenthesis ?

Reply:
This fixes PHP warnings about  non-existing index.